app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split()
+    com_from = command[1]
+    com_to = command[2]
+    if com_from[1] == com_to[2]:
```

Comment:
Why do you using indexes on str ? com_from it is already spited value . Also think more about viriables naming . `com_to , com_from` it is not very good naming inho